tools/analysis_tools/visualize_shape_bias.py

```diff
@@ -0,0 +1,244 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Modified from https://github.com/bethgelab/model-vs-human
+import argparse
+import os
+import os.path as osp
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from utils import FormatStrFormatter, ShapeBias
+
+# global default boundary settings for thin gray transparent
+# boundaries to avoid not being able to see the difference
+# between two partially overlapping datapoints of the same color:
+PLOTTING_EDGE_COLOR = (0.3, 0.3, 0.3, 0.3)
+PLOTTING_EDGE_WIDTH = 0.02
+ICONS_DIR = osp.join(osp.dirname(__file__), '../..', 'assets/icons')
```

Comment:
Better to avoid using `/` because of the windows system.
Besides, I think the 'icons' can be moved to `resources` folder and rename it called `shape_bias_icons`, it is no need to create a new `assets` folder.
```suggestion
ICONS_DIR = osp.join(osp.dirname(__file__), '..', '..', 'resources', 'shape_bias_icons')
```

Reply:
plan to remove there icons from the repo, and tell users to download them before plotting.